Sources/NIOConcurrencyHelpers/lock.swift

```diff
@@ -29,17 +29,21 @@ import Glibc
 /// `SRWLOCK` type.
 public final class Lock {
 #if os(Windows)
-    fileprivate let mutex: UnsafeMutablePointer<SRWLOCK> =
+    private let mutex: UnsafeMutablePointer<SRWLOCK> =
         UnsafeMutablePointer.allocate(capacity: 1)
+#elseif canImport(Darwin)
```

Comment:
We're using `#if os()` guards consistently for this, we should continue to do so.